ios/Classes/CountlyFlutterPlugin.m

```diff
@@ -1408,6 +1414,10 @@ - (void)populateConfig:(NSDictionary *)_config {
         if (loggingEnabled) {
             config.enableDebug = [loggingEnabled boolValue];
         }
+        NSNumber *locationDisabled = _config[@"locationDisabled"];
+        if (locationDisabled) {
```

Comment:
what happened if locationDisabled contains '0'?
I think the if check should be something like this" 
```
if (locationDisabled && [locationDisabled boolValue]) {
...
}
```

Reply:
done and pushed